README.md

```diff
@@ -68,6 +68,7 @@ class ExampleServiceClient
       error_threshold:  50,
 
       # seconds before the circuit times out
+      # if set to nil no timeout is used
```

Comment:
Make sense to me if we want to stick with the current naming I would suggest `0` as an indicator of no timeout this seems to be a common approach. I just really don't like to have `nil` here :). But yes I think you make a very good point about `:disabled` not being very intuitive.